src/LineageTree/lineageTree.py

```diff
@@ -1856,28 +1837,22 @@ def get_cycle(
             x, depth_succ
         )
 
-    @property
-    def all_tracks(self):
-        if not hasattr(self, "_all_tracks"):
-            self._all_tracks = self.get_all_tracks()
-        return self._all_tracks
-
-    def get_all_tracks(self, force_recompute: bool = False) -> list:
+    def get_all_tracks(self) -> list:
         """Computes all the tracks of a given lineage tree,
         stores it in `self.all_tracks` and returns it.
 
         Returns:
             ([[int, ...], ...]): list of lists containing track cell ids
         """
-        if not hasattr(self, "_all_tracks"):
-            self._all_tracks = []
+        if not hasattr(self, "all_tracks"):
+            self.all_tracks = []
             to_do = set(self.nodes)
             while len(to_do) != 0:
                 current = to_do.pop()
                 track = self.get_cycle(current)
-                self._all_tracks += [track]
+                self.all_tracks += [track]
                 to_do -= set(track)
-        return self._all_tracks
+        return self.all_tracks
```

Comment:
Here also, put it the way I did please.

You can remove the `force_recompute: bool = False`